lib/new_relic/agent/instrumentation/view_component/instrumentation.rb

```diff
@@ -0,0 +1,38 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module ViewComponent
+    INSTRUMENTATION_NAME = NewRelic::Agent.base_name(name)
+
+    def render_in_with_tracing(*args)
+      NewRelic::Agent.record_instrumentation_invocation(INSTRUMENTATION_NAME)
+
+      begin
+        segment = NewRelic::Agent::Tracer.start_segment(
+          name: metric_name(self.class.identifier, self.class.name)
+        )
+        yield
+      rescue => e
+        ::NewRelic::Agent.logger.debug('Error capturing ViewComponent segment', e)
```

Comment:
Late to the party, but... is suppressing errors here the intended behavior? 

https://github.com/ViewComponent/view_component/issues/1981